plugins/tiddlywiki/prosemirror/setup/keymap.js

```diff
@@ -19,59 +19,69 @@ var mac = typeof navigator != "undefined" ? /Mac|iP(hone|[oa]d)/.test(navigator.
 function buildKeymap(schema, mapKeys) {
 	var keys = {}, type;
 	function bind(key, cmd) {
-		if (mapKeys) {
+		if(mapKeys) {
 			var mapped = mapKeys[key];
-			if (mapped === false) return;
-			if (mapped) key = mapped;
+			if(mapped === false) return;
+			if(mapped) key = mapped;
 		}
 		keys[key] = cmd;
 	}
 
 	bind("Mod-z", prosemirrorHistory.undo);
 	bind("Shift-Mod-z", prosemirrorHistory.redo);
 	bind("Backspace", prosemirrorInputrules.undoInputRule);
-	if (!mac) bind("Mod-y", prosemirrorHistory.redo);
+	if(!mac) bind("Mod-y", prosemirrorHistory.redo);
 
 	bind("Alt-ArrowUp", prosemirrorCommands.joinUp);
 	bind("Alt-ArrowDown", prosemirrorCommands.joinDown);
 	bind("Mod-BracketLeft", prosemirrorCommands.lift);
 	bind("Escape", prosemirrorCommands.selectParentNode);
 
-	if (type = schema.marks.strong) {
+	type = schema.marks.strong;
+	if(type) {
 		bind("Mod-b", prosemirrorCommands.toggleMark(type));
 		bind("Mod-B", prosemirrorCommands.toggleMark(type));
 	}
-	if (type = schema.marks.em) {
+	type = schema.marks.em;
+	if(type) {
 		bind("Mod-i", prosemirrorCommands.toggleMark(type));
 		bind("Mod-I", prosemirrorCommands.toggleMark(type));
 	}
-	if (type = schema.marks.code)
+	type = schema.marks.code;
+	if(type)
 		bind("Mod-`", prosemirrorCommands.toggleMark(type));
-	if (type = schema.nodes.blockquote)
+	type = schema.nodes.blockquote;
+	if(type)
```

Comment:
if there is a linebreak after an if there has to be `{}` curly braces, otherwise the code will be super brittle to maintain. 
We should use curly braces also for 1 liners. There is no question if something will go wrong, only **when**.